package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "s53_worstratedmovies",
+  "version": "1.0.0",
+  "description": "",
```

Comment:
Add a meaningful description and author information in the package.json file. [medium]